packages/integration-tests-storybook-6_1_x/test/__snapshots__/bundle.integration.ts.snap

```diff
@@ -23,34 +23,34 @@ exports[`run bundle command 2`] = `
   \\"outputFilePath\\": \\"files/vendors.js\\"
  },
  {
-  \\"name\\": \\"Button.stories.js\\",
+  \\"name\\": \\"runtime~src__Button.stories.js.js\\",
```

Comment:
do we want to remove the `runtime~` prefix?

Reply:
That's automatically done by storybook. It's a chunk .The real Button.stories should be still there.